helm/observability-bundle/templates/apps.yaml

```diff
@@ -62,8 +62,8 @@ spec:
   {{- end }}
   {{- range $extraConfig := .extraConfigs }}
   - kind: {{ $extraConfig.kind }}
-    name: {{ $extraConfig.name }}
-    namespace: {{ $extraConfig.namespace }}
+    name: {{ $.Values.clusterID }}-{{ $extraConfig.name }}
+    namespace: {{ $.Release.Namespace }}
```

Comment:
Could we use those in the values instead?

Reply:
It's what I did no ?

Reply:
Yes, because at the end no more condition in my code !

Reply:
clusterId is coming from values, so I can't move it in the values

Reply:
I revert the last change ;-)